grader/flags.py

```diff
@@ -0,0 +1,83 @@
+from flagset import Flag, FlagSet
+
+fset = FlagSet({
+    "token": Flag(
+        str,
+        cmdline_name="token",
+        env_name="BROADWAY_TOKEN",
+        config_name="token",
+        help="cluster token",
+    ),
+    "grader_id": Flag(
+        str,
+        cmdline_name="grader-id",
+        env_name="BROADWAY_GRADER_ID",
+        config_name="id",
+        help="unique identifier of the grader instance",
+    ),
+    "api_host": Flag(
+        str,
+        default="http://127.0.0.1:1470",
+        cmdline_name="--api-host",
+        env_name="BROADWAY_API_HOST",
+        config_name="api_host",
+        help="API host. If the host starts with 'ws://' or 'wss://',"
+        + "grader will use the websocket endpoints",
+    ),
+    "verbose": Flag(
+        bool,
+        default=False,
+        cmdline_name=["-v", "--verbose"],
+        env_name="BROADWAY_VERBOSE",
+        config_name="verbose",
+        help="verbose mode",
+    ),
+    "heartbeat_interval": Flag(
```

Comment:
I think this would go in `grader/api_keys` because this is something set by the API and not an independent grader config option. `grader/api_keys` seems to hold all the stuff that the API decides.